File: source/code/python/authentication/mfa_get_account_factors1_resp.py
# In order to get all Factors
factors = account.factors

# Factors can be filtered by type. To retrieve only the Sms Factors you can do:
sms_factors = account.factors.search({'type': 'sms'})

# To retrieve only the Google Authenticator Factors you can do:
google_factors = account.factors.search({'type': 'google-authenticator'})

File: source/code/nodejs/authentication/mfa_get_account_factors1_resp.js
var account; // A previously fetched Account object

// In order to get all Factors
account.getFactors(function(err, collectionResource) {
  if (err) {
    return console.log(err);
  }

  console.log(collectionResource);
});

// Factors can be filtered by type. Change 'sms' to 'google-authenticator'
// to retrieve only the Google Authenticator Factors
var collectionQueryOptions = {
  type: 'sms'
};

account.getFactors(collectionQueryOptions, function(err, collectionResource) {
  if (err) {
    return console.log(err);
  }

  console.log(collectionResource);
});
